src/app/fdctl/run/topos/fd_firedancer.c

```diff
@@ -201,7 +200,7 @@ fd_topo_initialize( config_t * config ) {
   /**/                 fd_topob_link( topo, "pack_replay",  "pack_replay",  65536UL,                                  USHORT_MAX,                    1UL   );
   /**/                 fd_topob_link( topo, "poh_pack",     "replay_poh",   128UL,                                    sizeof(fd_became_leader_t) ,   1UL   );
 
-  /**/                 fd_topob_link( topo, "replay_voter", "replay_voter", 128UL,                                    FD_TPU_DCACHE_MTU,             1UL   );
+  /**/                 fd_topob_link( topo, "replay_voter", "replay_voter", 128UL,                                    FD_TXN_MAX_SZ,                 1UL   );
```

Comment:
Do you mean 1232 for this one? `FD_TPU_MTU`? I don't remember what the format for this link is, but I have a hard time imagining it's just an `fd_txn_t`.